kbchat/test_config.example.yaml

```diff
@@ -0,0 +1,20 @@
+# Rename this file to `test_config.yaml`
+
+config:
+  bots:
+    alice1:
+      # Alice should have an active Stellar account with a little bit of XLM in it
+      username: "alice"
+      paperkey: "foo bar car..."
+    }
```

Comment:
? What's the point of this close brace?

Reply:
copy/paste error from [tests.config.example.ts](https://github.com/keybase/keybase-bot/blob/master/__tests__/tests.config.example.ts).